xbmc/filesystem/NFSFile.cpp

```diff
@@ -143,14 +142,18 @@ struct nfs_context *CNfsConnection::getContextFromMap(const std::string &exportn
   if (it != m_openContextMap.end())
   {
     //check if context has timed out already
-    uint64_t now = XbmcThreads::SystemClockMillis();
-    if((now - it->second.lastAccessedTime) < CONTEXT_TIMEOUT || forceCacheHit)
+    auto now = std::chrono::steady_clock::now();
+    if (std::chrono::duration_cast<std::chrono::milliseconds>(now - it->second.lastAccessedTime)
+                .count() < CONTEXT_TIMEOUT ||
+        forceCacheHit)
     {
       //its not timedout yet or caller wants the cached entry regardless of timeout
       //refresh access time of that
       //context and return it
       if (!forceCacheHit) // only log it if this isn't the resetkeepalive on each read ;)
-        CLog::Log(LOGDEBUG, "NFS: Refreshing context for %s, old: %" PRId64", new: %" PRId64, exportname.c_str(), it->second.lastAccessedTime, now);
+        CLog::Log(LOGDEBUG, "NFS: Refreshing context for {}, old: %" PRId64 ", new: %" PRId64,
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)